### PR TITLE
Add manifest collection after pivot and rearrange the manifest directories

### PR DESCRIPTION
**Pull request by smoshiur1237**

### CODE REVIEW

**Comment by smoshiur1237** [comment]
/cc @kashifest 
/cc @tuminoid 
Here is the link to check the manifest before and after pivot, after re-pivot: https://jenkins.nordix.org/job/metal3_metal3_dev_env_release-1-3_integration_test_centos/269/

**Review by kashifest — DISMISSED** [review]

**Review by furkatgofurov7 — COMMENTED** [review]
/lgtm

/hold to avoid merging and give time for @tuminoid to review

File: tests/scripts/fetch_manifests.sh [hunk 1]
```diff
@@ -2,14 +2,25 @@
...
 
+# Initial manifest directory
 DIR_NAME="/tmp/manifests/bootstrap-before-pivot"
-
+DIR_NAME_AFTER_PIVOT="/tmp/manifests/target-after-pivot"
+DIR_NAME_AFTER_REPIVOT="/tmp/manifests/bootstrap-after-repivot"
 # Check if manifest directory exists
-if [ -d "${DIR_NAME}" ]; then
-  DIR_NAME="/tmp/manifests/bootstrap-after-repivot"
+if [[ -d "${DIR_NAME}" && -d "${DIR_NAME_AFTER_PIVOT}" ]]; then
```
Comment (tuminoid) [inline tests/scripts/fetch_manifests.sh:10]: `&&` goes outside the `[[ .. ]]`

```suggestion
if [[ -d "${DIR_NAME}" ]] && [[ -d "${DIR_NAME_AFTER_PIVOT}" ]]; then
```

File: tests/scripts/fetch_manifests.sh [hunk 2]
```diff
@@ -2,14 +2,25 @@
...
+  DIR_NAME="${DIR_NAME_AFTER_REPIVOT}"
+  mkdir -p "${DIR_NAME}"
+  # Ephemeral cluster kubeconfig
+  kconfig="/home/metal3ci/.kube/config"
+elif [[ -d "${DIR_NAME}" && ! -d "${DIR_NAME_AFTER_PIVOT}" ]]; then
```
Comment (tuminoid) [inline tests/scripts/fetch_manifests.sh:15]: `&&` goes outside the `[[ .. ]]`

```suggestion
elif [[ -d "${DIR_NAME}" ]] && [[ ! -d "${DIR_NAME_AFTER_PIVOT}" ]]; then
```

File: tests/scripts/fetch_manifests.sh [hunk 3]
```diff
@@ -2,14 +2,25 @@
...
+  DIR_NAME="${DIR_NAME_AFTER_PIVOT}"
   mkdir -p "${DIR_NAME}"
+  # Target cluster kubeconfig
+  kconfig=$(sudo find /tmp/ -type f -name "kubeconfig*")
```
Comment (tuminoid) [inline tests/scripts/fetch_manifests.sh:19]:
```suggestion
  kconfig="$(sudo find /tmp/ -type f -name "kubeconfig*")"
```

**Review by tuminoid — COMMENTED** [review]
Minor nits

**Review by tuminoid — COMMENTED** [review]
/lgtm

**Pull request merged**